### DISCUSSION TITLE
Is there a way to search by partial text with by.text? Thx!

### DISCUSSION THREAD

**Discussion by mariushorvat** [wix/Detox#3923] [General]
I wanted to use this matcher with a partial text (eg. beginswith, contains, endswith, like, matches). Does it take this parameters?
https://wix.github.io/Detox/docs/api/matchers#bytexttext

**Comment by noomorph** [comment]
There was some work here: https://github.com/wix/Detox/issues/3366
@asafkorem did we push any drafts to a branch?
